packages/browser/src/login/oidc/TokenRequester.ts

```diff
@@ -96,18 +96,15 @@ export default class TokenRequester {
       throw new Error(`This issuer [${issuer}] does not have a token endpoint`);
     }
 
-    await this.dpopClientKeyManager.generateClientKeyIfNotAlready();
+    const key = await generateJwkForDpop();
```

Comment:
Just a suggestion, but perhaps a nice opportunity to expand the JWK acronym here, making the code slightly more readable, i.e.
`const jsonWebKey = await generateJwkForDpop();`

Reply:
Fixed in bbf7d447ecb4e6ee1b180e8cf7d839c3c991e9b9